README.md

```diff
@@ -112,6 +112,11 @@ By default, redis and sentinel will be called with de basic command, giving the
 
 If necessary, this command can be changed with the `command` option inside redis/sentinel spec. An example can be found in the [custom command example file](example/redisfailover/custom-command.yaml).
 
+### Custom Pod Annotations
+By default, no pod annotations will be applied to the Redis or Sentinel pods.
+
+In order to apply custom pod Annotations, you can provide the `podAnnotation` option inside redis/sentinel spec. An example can be found in the [custom annotations example file](example/redisfailover/custom-annotations.yaml).
```

Comment:
* I think `podAnnotation` should be `podAnnotations` as used in `example/redisfailover/custom-annotations.yaml` file.
* Rephrase a little to avoid redundant connector, i.e.: In order to apply custom pod Annotations, you can provide `podAnnotation` option inside redis/sentinel spec. An example can be found at [custom annotations example file](example/redisfailover/custom-annotations.yaml).

Reply:
@marcemq 
Nice catch! Totally missed that trailing s.

To be consistent with the other examples in the README, I used this wording since it is also found in the `Custom shutdown script` and `Custom command` sections. Would you want me to update wording there as well for consistency?

Reply:
Not sure, maybe the other reviewers can guide us on this.

Reply:
@slok @ese @jchanam 
Pinging to follow up on this

Reply:
@slok @ese @jchanam 

Bumping this again